Updates to Android 12 for some applications (#159)
riverar/wsa-app-compatibility: README.md

| Fate/Grand Order (US) FGO | 2.22.1 (135) | 11 | ✅ || A little unstable, but playable
| Files by Google | Unknown | Android 11 | ✅ || Works fine
| Fire Emblem Heroes | 6.0.0 | 11 | 🆖 | Requires GMS. If GMS is installed, it cannot be played due to SafetyNet error.

> | Firefox | 101.2.0 (2015885319) | 11/12 | ⚠️ | On Android 11, it works albeit with improperly rendered webpages. On Android 12, webpages doesn't render anymore causing to consume more resources | Tested only on IntelHD integrated graphics (I'm not sure for AMD or NVIDIA GPUs since I don't have it on hand).

Reviewer: ```suggestion
| Firefox | 101.2.0 (2015885319) | 12 & 11 | ⚠️ | On Android 11, it works albeit with improperly rendered webpages. On Android 12, webpages doesn't render anymore causing to consume more resources | Tested only on Intel HD integrated graphics.
```

| Sword Art Online: Unleash Blading | 3.2.0 | 11 | ⚠️ | Can't detect device
| Symbolab | 9.3.0 | 11 | ✅ || Keyboard not working, in-app keyboard is available though
| Sync for Reddit Pro | 20.0.3 | 11 | ✅
| Tachiyomi | 0.13.4 | 11/12 | ✅
| TachiyomiJ2K/TachiJ2K | 1.5.4 | 11/12 | ✅ | Parsing links (from a browser) causes to open the Tachiyomi extension window instead of opening TachiJ2K itself.
| Teamfight Tactics | 12.5.4259171 | 11 | ⚠️ | Crashes often before getting in game but after getting in, not many issues. Can get laggy at times but somewhat playable.
| TeamViewer | 15.22.136 | 11 | ✅
| Telegram | 8.1.2 | 11 | ✅

> | Termux (F-droid) | 0.118.0 | 11/12 |✅ || Termux (Play Store) version is depricated since Nov-02-2020

Reviewer: ```suggestion
| Termux (F-droid) | 0.118.0 | 12 & 11 |✅
```
